Common/src/main/resources/assets/hexcasting/lang/en_us.flatten.json5

```diff
@@ -412,6 +412,10 @@
     shame: "Shame on %s!",
   },
 
+  "death.attack.hexcasting.overcast": {
+    player: "%s's mind was subsumed into energy while fighting %s"
+  },
```

Comment:
This could be moved to the above block, something like this:
```json5
  "death.attack.hexcasting": {
    overcast: {
      "": "%s's mind was subsumed into energy",
      player: "%s's mind was subsumed into energy while fighting %s",
    },
    shame: "Shame on %s!",
  },
```